src/main/reactjs/public/i18n/en-GB.json

```diff
@@ -1 +1,12 @@
-{}
+{
+  "akt": {
+    "header": {
+      "title": "AKT",
+      "lang": {
+        "fi": "Suomeksi",
+        "sv": "På Svenska",
+        "en": "In English"
```

Comment:
Title casen sijaan voisi olla kielten kapitalisaatiosääntöjen mukaan? -> `suomeksi, på svenska, in English`.
Perusmuotoiset `suomi, svenska, English` voisi kanssa olla ihan toimiva vaihtoehto.

Reply:
Tuossa otettu mallia oph.fi:stä.